static.json

```diff
@@ -47,7 +47,7 @@
             "url": "/sensu-go/latest/operations/control-access/create-read-only-user",
             "status": 301
         },
-        "~ ^/sensu-go/6.3/?((?<=\/).*)?$": {
+        "~ ^/sensu-go/6.4/?((?<=\/).*)?$": {
             "url": "/sensu-go/latest/$1",
             "status": 302
         },
```

Comment:
```suggestion
        },
        "~ ^/sensu-go/6.0/?((?<=\/).*)?$": {
            "url": "/sensu-go/latest/$1",
            "status": 301
        },
```

Reply:
This regex will redirect any 6.0 links to whatever is the `latest` version forevermore